Tests/ConfigurationServiceSpec.swift

```diff
@@ -33,7 +33,8 @@ class ConfigurationServiceSpec: QuickSpec {
                         done()
                     }
                 }
-                expect(httpSession.sentRequest?.url).to(equal(configURL))
+                expect(httpSession.sentRequest?.url?.scheme).to(equal(configURL.scheme))
+                expect(httpSession.sentRequest?.url?.host).to(equal(configURL.host))
```

Comment:
`configURL` has query items as it is now a GET request.
So `scheme` (http) and `host` (config.url) are used to match the previous test (http://config.url).